public/components/trace_analytics/requests/dashboard_request_handler.ts

```diff
@@ -189,7 +190,12 @@ export const handleJaegerDashboardRequest = async (
       });
       return map;
     })
-    .catch((error) => console.error(error));
+    .catch((error) => {
+      console.log("error here")
+      console.error(error)
+      
+      setToast('hello')
```

Comment:
replace with appropriate error message from PM/UX

Reply:
@derek-ho @joshuali925 @ps48 

This almost shipped. This PR should not have been merged without addressing this problem.